docs/getting-started.md

```diff
@@ -90,6 +90,8 @@ $ tar Cxzvf /opt/cni/bin cni-plugins-linux-amd64-v1.1.1.tgz
 
 The binaries are built statically and should work on any Linux distribution.
 
+In order for containerd to use them, you need to create a `.conflist` file in `/etc/cni/net.d`. See [install-cni](https://github.com/containerd/containerd/blob/main/script/setup/install-cni) for a basic config.
```

Comment:
I did some more research, this is PEBKAC.
The netconf seems like it is supposed to be configured by either
* (for k8s) the network add-on - https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network
* custom network configuration made by the user of the CNI plugin https://github.com/containernetworking/cni?tab=readme-ov-file#running-the-plugins

Is there a precise way to mention this?